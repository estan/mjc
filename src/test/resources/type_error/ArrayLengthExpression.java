class ArrayLengthExpressionTest {
    public static void main(String[] args) {
        int i;
        int[] ia;
        long l;
        long[] la;
        boolean b;
        A cA;
        B cB;

        i = i.length;  // LENGTH_ON_NON_ARRAY_TYPE
        i = l.length;  // LENGTH_ON_NON_ARRAY_TYPE
        i = b.length;  // LENGTH_ON_NON_ARRAY_TYPE
        i = cA.length; // LENGTH_ON_NON_ARRAY_TYPE
    }
}
class A {}
class B {}
