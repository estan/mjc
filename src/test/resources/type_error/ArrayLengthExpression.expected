LENGTH_ON_NON_ARRAY_TYPE
LENGTH_ON_NON_ARRAY_TYPE
LENGTH_ON_NON_ARRAY_TYPE
LENGTH_ON_NON_ARRAY_TYPE
